lib/exception_hunter/error_creator.rb

```diff
@@ -15,6 +16,12 @@ class << self
       def call(tag: nil, **error_attrs)
```

Comment:
The name `call` doesn't make sense anymore, does it? Perhaps `create_or_schedule_create`? I don't like it either, open to suggestions

Reply:
We could also do something like 
```ruby
def call(async: Config.async, tag: nil, ...)
```

Which allows us to inject the config always giving us more flexibility. Your call anyway